### ISSUE TITLE
concurrent execution of test cases in folders

### ISSUE THREAD

**Issue by azplanlos** [rangav/thunder-client-support#1400]
**Please describe feature/problem details and the solution you'd like.**
We have test collections organizing test cases consisting of multiple requests within folders. Due to performance optimizations in our build pipeline we want to run multiple folders concurrently while the requests within each folder need to be send sequentially. Is there a way to achieve this or a possibility for you to implement this?

**Describe alternatives you've considered**
- Splitting up into different collections - not an option as there are too many test cases
- Chaining the requests - not an option as there are too many requests and will pollute the collection

**Are you using the free version/paid version/trial:**
paid version

**Comment by rangav** [comment]
@azplanlos Thanks for the feedback.

Will research and get back to you.

Did you try the [Run Collection Parallel](https://github.com/rangav/thunder-client-support/releases/tag/v2.15.0) feature? is it useful?

**Comment by azplanlos** [comment]
Our collections are too big to be split up, we would end up with hundrets of collections.

**Comment by rangav** [comment]
You dont need to split collection. The feature allows you to run multiple requests in parallel from single collection.

**Comment by azplanlos** [comment]
yes, but the actual requests within the same folder need to be run sequential.

**Comment by rangav** [comment]
Ok got it. Will think of a solution.

Will command something like this will work?

`tc --fol "Fol1,Fol2,Fol3" --concurrent`

Do you want all folders to run in parallel or you want to configure the of folders e.g `--concurrent 3`

**Comment by rangav** [comment]
Thanks @azplanlos for the feedback

**Comment by azplanlos** [comment]
Hi @rangav, is there any plan to implement this in the near future?

**Comment by rangav** [comment]
@azplanlos currently not in the roadmap.
